docs/concepts/what-is-a-temporal-cluster.md

```diff
@@ -69,3 +69,48 @@ To enhance this feature, Temporal supports an [integration with Elasticsearch](/
 - Elasticsearch v7.10 is supported from Temporal version 1.7.0 onwards
 - Elasticsearch v6.8 is supported in all Temporal versions
 - Both versions are explicitly supported with AWS Elasticsearch
+
+### mTLS Encryption
+
+Temporal supports Mutual Transport Layer Security (mTLS) as a method of encrypting network traffic between services within a Temporal Cluster, or between application processes and a Cluster.
+
+Mutual TLS can be enabled in Temporal’s [TLS configuration](/references/configuration#tls). This configuration can be passed along through WithConfig or WithConfigLoader.
```

Comment:
Can this also be updated with Dynamic Config ?

Reply:
Unsure of this